mkosi/manifest.py

```diff
@@ -197,6 +199,31 @@ def record_deb_packages(self, root: Path) -> None:
 
             source_package.add(package)
 
+    def record_pkg_packages(self, root: Path) -> None:
+        package_descriptions = [str(p) for p in Path("/var/lib/pacman/local").glob("*/desc")]
```

Comment:
Shouldn't we be looking inside `root` here? This will operate on the host system no?

Reply:
Argh, I didn't tripple check the path. You are correct!

Reply:
Sorry, my bad, copied that from my local test.